config/etc/system-image/writable-paths

```diff
@@ -94,7 +78,9 @@
 /etc/systemd/system.conf.d              auto                    persistent  transition  none
 /etc/systemd/user.conf.d                auto                    persistent  transition  none
 /etc/systemd/logind.conf.d              auto                    persistent  transition  none
+# FIXME: what is this used for?
 /etc/iproute2                           auto                    persistent  transition  none
+# FIXME: why do we need this in the systemd age?
 /etc/rc0.d                              auto                    persistent  transition  none
```

Comment:
from debian/changelog:

  * add all /etc/rc*.d directories to writable paths to quieten systemctl
disable/enable (we dont use these dirs anyway but they cause warnings)